projects/storefrontapp-e2e-cypress/cypress/integration/regression/my-account/payment-methods.e2e-spec.ts

```diff
@@ -7,21 +8,16 @@ import {
   visitPaymentDetailsPage,
 } from '../../../helpers/payment-methods';
 import { viewportContext } from '../../../helpers/viewport-context';
-import {
-  addProductToCartViaAutoComplete,
-  addProductToCartViaSearchPage,
-  loginRegisteredUser,
-} from '../../../helpers/cart';
 
 describe('Payment Methods', () => {
-  viewportContext(['mobile', 'desktop'], () => {
+  viewportContext(['mobile'], () => {
```

Comment:
Do we want to run it only on mobile or just a leftover?

Reply:
GAH leftover. Fixed.